<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCourseDialog</class>
 <widget class="QDialog" name="AddCourseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>新建课程</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #CCCCFF;
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QLabel" name="coursetype">
     <property name="text">
      <string>课程性质：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QTextEdit" name="precourses_val"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="credit">
     <property name="text">
      <string>学分：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="name">
     <property name="text">
      <string>课程名称：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="precourses">
     <property name="text">
      <string>先修课程：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="credit_val"/>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="name_val">
     <property name="styleSheet">
      <string notr="true">selection-color: rgb(85, 255, 127);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="coursetype_val">
     <item>
      <property name="text">
       <string>必修课程</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>选修课程</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="save_btn">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="cancel_btn">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
